### PR TITLE
(enhancement)type: add support for mounting files on host

### PR DESCRIPTION
**Pull request by ksatchit**
Signed-off-by: ksatchit <karthik.s@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes # (partially fulfills https://github.com/litmuschaos/litmus/issues/1650) 

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag
-   [ ] PR messages has document related information
-   [ ] Labelled this PR & related issue with `breaking-changes` tag
-   [ ] PR messages has breaking changes related information
-   [ ] Labelled this PR & related issue with `requires-upgrade` tag
-   [ ] PR messages has upgrade related information
-   [ ] Commit has unit tests
-   [ ] Commit has integration tests

### CODE REVIEW

File: pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go [hunk 1]
```diff
@@ -52,6 +52,12 @@ type Secret struct {
 	MountPath string `json:"mountPath"`
 }
 
+type HostFile struct {
+	Name      string `json:"name"`
+	MountPath string `json:"mountPath"`
+    NodePath  string `json:"nodePath"`
```
Comment (rahulchheda) [inline pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go:58]: please fix spaces here

Comment (chandankumar4) [inline pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go:58]: Some indentation issue is here also in line: 85

File: pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go [hunk 2]
```diff
@@ -75,6 +81,8 @@ type ExperimentDef struct {
 	ConfigMaps []ConfigMap `json:"configmaps,omitempty"`
 	// Secrets contains a list of Secrets
 	Secrets []Secret `json:"secrets,omitempty"`
+    // HostPathVolume defines the host directory/file to be mounted
```
Comment (rahulchheda) [inline pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go:84]: Here too.

File: pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go (hunk 1 above)
Comment (ksatchit) [reply pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go]: fixed!

Comment (ksatchit) [reply pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go]: done!

File: pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go (hunk 2 above)
Comment (ksatchit) [reply pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go]: done!

File: pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go [hunk 3]
```diff
@@ -75,6 +81,8 @@ type ExperimentDef struct {
 	ConfigMaps []ConfigMap `json:"configmaps,omitempty"`
 	// Secrets contains a list of Secrets
 	Secrets []Secret `json:"secrets,omitempty"`
+	// HostPathVolume defines the host directory/file to be mounted
```
Comment (rahulchheda) [inline pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go:84]: nit: Shouldn't this be HostFileVolume ?

Comment (ksatchit) [reply pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go]: fixed!

**Comment by ksatchit** [comment]
Thanks for the reviews! Merging the PR on account on previous approvals (the last change was a comment fix)

**Pull request merged**